database/ServiceMySql.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace database;
+
+use exceptions\EmptyTableException;
+use exceptions\InvalidIdException;
+use src\Student;
+
+class ServiceMySql implements ServiceInterface
+{
+	private $conn;
+
+	public function __construct($conn)
```

Comment:
typehint what type is $conn (as PHP is not strongly typed, this is not necessary, but highly recommended)